docs/docs/1.7.x/security/api-server-auth.md

```diff
@@ -1,156 +1,167 @@
-# API Server Authentication
+# API server authentication
 
-By default, Kuma exposes API Server on [ports](../networking/networking.md) `5681` and `5682` (protected by builtin TLS). This server is used for actions like
-* Accessing policies and objects
-* Managing policies and objects (on Universal)
-* Accessing GUI
+Kuma exposes API server on [ports](../networking/networking.md) `5681` and `5682` (protected by TLS).
 
-Authenticated user can be authorized to execute administrative actions such as
-* Managing Kuma Secrets (on Universal)
-* Generating Dataplane Tokens
-* Generating User Tokens
+An authenticated user can be authorized to execute administrative actions such as
+* Managing administrative resources like Kuma Secrets on Universal
+* Generating user token, data plane proxy token, zone ingress token, zone token
 
-## User Token
+## User token
 
-User Token is a JWT token that contains
+A user token is a signed JWT token that contains
 * The name of the user
-* The list of groups that user belongs to
+* The list of groups that a user belongs to
 * Expiration date of the token
 
-The User Token is signed by a signing key that is autogenerated on the control plane. The signing key is SHA256 encrypted.
+### Groups
 
-You can check for the signing key:
+A user can be a part of many groups. Kuma adds two groups to a user automatically:
+* Every authenticated user is a part of `mesh-system:authenticated`.
+* Every user that doesn't provide authentication data is a part of `mesh-system:unauthenticated`.
```

Comment:
Why do we need to explain what unauthenticated (user that doesn't provide authentication) means? Is it ok to just make it simpler by doing something like this:

```suggestion
* authenticated users are a part of `mesh-system:authenticated`.
* unauthenticated users are a part of `mesh-system:unauthenticated`.
```